.travis.yml

```diff
@@ -1,13 +1,18 @@
 sudo: required
 language: node_js
 node_js:
-  - "0.10.25"
+  - "6"
```

Comment:
@paulkerschen this might be a config trick I don't know. Does "6" give us a snapshot of the latest stable 6.* release?  I would expect "6.5.0" here.